### PR TITLE
Testnet documentation update

### PR DESCRIPTION
**Pull request by measure-fi**
https://github.com/palomachain/paloma/issues/143

# Background

Organize our testnet setup documentation better, explicitly add connection strings, and generally
address retrospective items from previous testnet deploy.

# Testing completed

- [ ] Manual review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,19 +61,19 @@ MONIKER="$(hostname)"
...
 
-Copy several files from the original server to the new server:
-  - `.paloma/config/app.toml`
-  - `.paloma/config/client.toml`
-  - `.paloma/config/config.toml`
-  - `.paloma/config/genesis.json`
+Copy the `genesis.json` file of the testnet we wish to connect to
+
+```shell
+wget -O .paloma/config/genesis.json https://raw.githubusercontent.com/palomachain/paloma/chase-doc/doc/genesis.json
```
Comment (measure-fi) [inline README.md:67]: `s/chase-doc/master/` before final push

File: README.md [hunk 2]
```diff
@@ -131,9 +106,46 @@ palomad tx staking create-validator \
...
 ```shell
+MAIN_PEER_DESIGNATION=da10f3078f78dabbb38b59df2c90230716f3149c@157.245.76.119:26656
 palomad start --p2p.persistent_peers "$MAIN_PEER_DESIGNATION"
 ```
+
+### Uploading a local contract
+
+```shell
+CONTRACT=<contract.wasm>
+palomad tx wasm store $CONTRACT --from my_validator --broadcast-mode block -y --gas auto --fees 3000dove
```
Comment (measure-fi) [inline README.md:120]: my_validator cannot be hard coded here.

File: doc/genesis.json [hunk 3]
```diff
@@ -0,0 +1,295 @@
+{
```
[the remark this answers is not in the document]
Comment (Vizualni) [reply doc/genesis.json:1]: Yes. This should live in a separate repository (e..g https://github.com/terra-money/classic-mainnet/tree/master/columbus-1)

[the remark this answers is not in the document]
Comment (measure-fi) [reply doc/genesis.json:1]: 👍 
That's mainnet, but they also have one for testnet files https://github.com/terra-money/testnet.

[the remark this answers is not in the document]
Comment (measure-fi) [reply doc/genesis.json]: Created https://github.com/palomachain/testnet

File: README.md [hunk 4]
```diff
@@ -86,28 +61,28 @@ MONIKER="$(hostname)"
...
 
-Copy several files from the original server to the new server:
-  - `.paloma/config/app.toml`
-  - `.paloma/config/client.toml`
-  - `.paloma/config/config.toml`
-  - `.paloma/config/genesis.json`
+Copy the `genesis.json` file of the testnet we wish to connect to
+
+```shell
+wget -O .paloma/config/genesis.json https://raw.githubusercontent.com/palomachain/testnet/master/columba/genesis.json
```
Comment (Vizualni) [inline README.md:67]: @measure-fi could we pick another name which doesn't sound like it was taken directly from terra? :)

File: README.md [hunk 5]
```diff
@@ -39,44 +39,19 @@ See [Release procedure](CONTRIBUTING.md#release-procedure) for more information
 
 N/A
 
-## Install
+## Testnet
```
Comment (Vizualni) [inline README.md:42]: Please rename this to something like "Temporary testnet setup instructions". These steps is not how it's going to work for the real testnet. This was setup that I wrote for our local docker setup which is not how it should work in real world.

Comment (measure-fi) [reply README.md]: Agreed, although that reads to me as if the testnet is meant to be temporary. I have changed it to "Testnet Setup Instructions"

File: README.md (hunk 4 above)
Comment (measure-fi) [reply README.md]: Renamed to devnet.

File: README.md [hunk 6]
```diff
@@ -86,28 +61,28 @@ MONIKER="$(hostname)"
...
 
-Copy several files from the original server to the new server:
-  - `.paloma/config/app.toml`
-  - `.paloma/config/client.toml`
-  - `.paloma/config/config.toml`
-  - `.paloma/config/genesis.json`
+Copy the `genesis.json` file of the testnet we wish to connect to
+
+```shell
+wget -O .paloma/config/genesis.json https://raw.githubusercontent.com/palomachain/devnet/master/columba/genesis.json
```
[the remark this answers is not in the document]
Comment (measure-fi) [reply README.md:67]: Oh. Ha. Yes I will pick a different pigeon.

**Comment by measure-fi** [comment]
@Vizualni PTAL

File: README.md [hunk 7]
```diff
@@ -86,28 +61,28 @@ MONIKER="$(hostname)"
...
 
-Copy several files from the original server to the new server:
-  - `.paloma/config/app.toml`
-  - `.paloma/config/client.toml`
-  - `.paloma/config/config.toml`
-  - `.paloma/config/genesis.json`
+Copy the `genesis.json` file of the testnet we wish to connect to
+
+```shell
+wget -O .paloma/config/genesis.json https://raw.githubusercontent.com/palomachain/devnet/master/livia/genesis.json
```
Comment (Vizualni) [inline README.md:67]: Lets rename the devnet to testnet as well.

**Review by Vizualni — APPROVED** [review]
LGTM with a comment

**Pull request merged**